### PR TITLE
explain how to skip rows between header & data

### PR DESCRIPTION
**Pull request by dacoex**
This is the result from
https://github.com/pydata/pandas/issues/10898

### CODE REVIEW

File: doc/source/cookbook.rst [hunk 1]
```diff
@@ -971,6 +971,54 @@ Parsing date components in multi-columns is faster with a format
...
+    01.01.1990 02:00;9;5;6;7
+    01.01.1990 03:00;13;7;8;9
+    01.01.1990 04:00;17;9;10;11
+    01.01.1990 05:00;21;11;12;13
+    """
+    
+Option 1: pass rows explicitly to skiprows
+""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: python
```
Comment (jreback) [inline doc/source/cookbook.rst]: do this in an ipython block

File: doc/source/cookbook.rst [hunk 2]
```diff
@@ -971,6 +971,54 @@ Parsing date components in multi-columns is faster with a format
...
+
+    pd.read_csv(StringIO(data.decode('UTF-8')), sep=';', skiprows=[11,12], index_col=0, parse_dates=True, header=10)
+
+    
+Option 2: read column names and then data
+""""""""""""""""""""""""""""""""""""
+
+.. code-block:: python
```
Comment (jreback) [inline doc/source/cookbook.rst]: same here

File: doc/source/cookbook.rst [hunk 3]
```diff
@@ -971,6 +971,54 @@ Parsing date components in multi-columns is faster with a format
...
+    
+Option 1: pass rows explicitly to skiprows
+""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: python
+
+    pd.read_csv(StringIO(data.decode('UTF-8')), sep=';', skiprows=[11,12], index_col=0, parse_dates=True, header=10)
+
+    
+Option 2: read column names and then data
```
Comment (jreback) [inline doc/source/cookbook.rst]: this has to be the same length as the header line

File: doc/source/cookbook.rst [hunk 4]
```diff
@@ -971,6 +971,54 @@ Parsing date components in multi-columns is faster with a format
...
+    01.01.1990 04:00;17;9;10;11
+    01.01.1990 05:00;21;11;12;13
+    """
+    
+Option 1: pass rows explicitly to skiprows
+""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: python
+
+    pd.read_csv(StringIO(data.decode('UTF-8')), sep=';', skiprows=[11,12], index_col=0, parse_dates=True, header=10)
```
Comment (jreback) [inline doc/source/cookbook.rst]: don't make the lines so long, wrap them

File: doc/source/cookbook.rst [hunk 5]
```diff
@@ -971,6 +971,54 @@ Parsing date components in multi-columns is faster with a format
...
+        ;m²;°C;m²;m
+    ;;;;
+    01.01.1990 00:00;1;1;2;3
+    01.01.1990 01:00;5;3;4;5
+    01.01.1990 02:00;9;5;6;7
+    01.01.1990 03:00;13;7;8;9
+    01.01.1990 04:00;17;9;10;11
+    01.01.1990 05:00;21;11;12;13
+    """
+
```
Comment (jreback) [inline doc/source/cookbook.rst]: you need an ``ipython:: python`` block here (for each of these options)

**Comment by jreback** [comment]
can you update

**Comment by dacoex** [comment]
>can you update

What is missing?

File: doc/source/cookbook.rst [hunk 6]
```diff
@@ -971,6 +971,54 @@ Parsing date components in multi-columns is faster with a format
...
+    01.01.1990 02:00;9;5;6;7
+    01.01.1990 03:00;13;7;8;9
+    01.01.1990 04:00;17;9;10;11
+    01.01.1990 05:00;21;11;12;13
+    """
+    
+    Option 1: pass rows explicitly to skiprows
+    """"""""""""""""""""""""""""""""""""""""""
+
+    pd.read_csv(StringIO(data.decode('UTF-8')), sep=';', skiprows=[11,12],
```
Comment (jorisvandenbossche) [inline doc/source/cookbook.rst]: You need start a new `.. ipython:: python` here

File: doc/source/cookbook.rst [hunk 7]
```diff
@@ -971,6 +971,54 @@ Parsing date components in multi-columns is faster with a format
...
+    01.01.1990 00:00;1;1;2;3
+    01.01.1990 01:00;5;3;4;5
+    01.01.1990 02:00;9;5;6;7
+    01.01.1990 03:00;13;7;8;9
+    01.01.1990 04:00;17;9;10;11
+    01.01.1990 05:00;21;11;12;13
+    """
+    
+    Option 1: pass rows explicitly to skiprows
+    """"""""""""""""""""""""""""""""""""""""""
```
Comment (jorisvandenbossche) [inline doc/source/cookbook.rst]: The header should not be indented

**Comment by jreback** [comment]
@jorisvandenbossche what do you think?

**Comment by jorisvandenbossche** [comment]
Seems OK! @dacoex Could you squash the commits? (http://pandas-docs.github.io/pandas-docs-travis/contributing.html#combining-commits)

**Comment by dacoex** [comment]
@jorisvandenbossche  done / see above.

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**